server/src/main/java/org/elasticsearch/action/main/MainResponse.java

```diff
@@ -105,6 +117,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         builder.field("name", nodeName);
         builder.field("cluster_name", clusterName.value());
         builder.field("cluster_uuid", clusterUuid);
+        if (!Strings.isEmpty(clusterDynamicName)) {
```

Comment:
`false == Strings` is our style and I've grown to rather like it! I find it a little more obvious.